src/Components/Fixtures/index.ts

```diff
@@ -29,3 +29,92 @@ export const proposalStatusPropsMock = {
   status: ContractState.Active,
   endTime: moment('2022-05-09T08:00:00'),
 };
+

```

Comment:
Pretty sure we have fixture files in each component folder just like tests and snapshots, lets keep it consistent